nats-server/nats_admin/signals.md

```diff
@@ -5,7 +5,9 @@ On Unix systems, the NATS server responds to the following signals:
 | Signal | Result |
 | :--- | :--- |
 | `SIGKILL` | Kills the process immediately |
+| `SIGQUIT` | Kills the process immediately and performs a core dump |
 | `SIGINT` | Stops the server gracefully |
+| `SIGTERM` | Stops the server gracefully |
```

Comment:
There are two signals that do the same thing? 'SIGINT' = 'SIGTERM'

Reply:
yes they are the same